frontend/pages/post/[id].tsx

```diff
@@ -22,10 +22,14 @@ const PostPage: NextPage = () => {
       <LoadingWrapper
         loading={loading}
         error={error}
-        render={() => <Post post={post} refetch={refetch} />}
+        render={() => <NewPost post={post} refetch={refetch} />}
```

Comment:
Sorry I'm a bit confused about how this makes sense 🤔

Reply:
Doh! I think I confused myself ! I will revert this back to `Post` and then make the Posts that appear on the my feed / profile / my posts pages `PostCard`